images/temporal-admin-tools/config/template.apko.yaml

```diff
@@ -1,19 +1,29 @@
 contents:
-  packages: [
-    # Package "tctl" comes in via var.extra_packages
-    # To add a version stream image, see the extra_packages variable in config/main.tf to add packages conditionally.
-    # Otherwise, just add packages here.
-  ]
+  packages:
+    - tini
 
 accounts:
   groups:
-    - groupname: nonroot
-      gid: 65532
+    - groupname: temporal
+      gid: 1000
   users:
-    - username: nonroot
-      uid: 65532
-      gid: 65532
-  run-as: 65532
+    - username: temporal
+      uid: 1000
+      gid: 1000
+  run-as: 1000
+
+paths:
+  - path: /etc/temporal
+    type: directory
+    uid: 1000
+    gid: 1000
+    permissions: 0o755
+    recursive: true
+
+environment:
+  TEMPORAL_HOME: /etc/temporal
 
 entrypoint:
-  command: /usr/bin/tctl
+  command: /sbin/tini -- sleep infinity
```

Comment:
The `sleep` bit here will require the `busybox` package.

Reply:
Refactored tests a bit and used busybox so that sleep is available, thanks!